src/components/Tabs/tests/Tabs.test.tsx

```diff
@@ -193,4 +206,214 @@ describe('<Tabs />', () => {
+      const tabs = mountWithAppProvider(<Tabs {...mockProps} />);
+      expect(tabs.find(TabMeasurer).prop('tabToFocus')).toBe(-1);
+    });
+
+    it('resets focus when selected gets updated to the previously selected value', () => {
+      const tabs = mountWithAppProvider(
+        <Tabs {...mockProps} selected={0} tabs={mockTabs} />,
+      );
+      trigger(tabs.find('ul'), 'onKeyUp', {
+        key: 'ArrowRight',
+      });
+      tabs.setProps({selected: 0});
+      expect(tabs.find(TabMeasurer).prop('tabToFocus')).toBe(-1);
+    });
+
+    it('passes the correct selected value if given', () => {
+      const tabs = mountWithAppProvider(
+        <Tabs {...mockProps} selected={1} tabs={mockTabs} />,
+      );
+      expect(tabs.find(TabMeasurer).prop('selected')).toBe(1);
+    });
+
+    describe('ArrowRight', () => {
+      it('shifts focus to the next tab when pressing ArrowRight', () => {
+        const tabs = mountWithAppProvider(
+          <Tabs {...mockProps} tabs={mockTabs} />,
+        );
+        trigger(tabs.find('ul'), 'onKeyUp', {
+          key: 'ArrowRight',
+        });
+        expect(tabs.find(TabMeasurer).prop('tabToFocus')).toBe(0);
+      });
+
+      it('shifts focus to the first tab when pressing ArrowRight on the last tab', () => {
+        const tabs = mountWithAppProvider(
+          <Tabs {...mockProps} tabs={mockTabs} />,
+        );
+        let x;
+        for (x = 0; x <= 3; x++) {
+          trigger(tabs.find('ul'), 'onKeyUp', {
+            key: 'ArrowRight',
+          });
+        }
+        expect(tabs.find(TabMeasurer).prop('tabToFocus')).toBe(0);
+      });
+    });
+
+    describe('ArrowDown', () => {
+      it('shifts focus to the next tab when pressing ArrowDown', () => {
+        const tabs = mountWithAppProvider(
+          <Tabs {...mockProps} tabs={mockTabs} />,
+        );
+        trigger(tabs.find('ul'), 'onKeyUp', {
+          key: 'ArrowDown',
+        });
+        expect(tabs.find(TabMeasurer).prop('tabToFocus')).toBe(0);
+      });
+
+      it('shifts focus to the first tab when pressing ArrowDown on the last tab', () => {
+        const tabs = mountWithAppProvider(
+          <Tabs {...mockProps} tabs={mockTabs} />,
+        );
+        let x;
+        for (x = 0; x <= 3; x++) {
+          trigger(tabs.find('ul'), 'onKeyUp', {
+            key: 'ArrowDown',
+          });
+        }
+        expect(tabs.find(TabMeasurer).prop('tabToFocus')).toBe(0);
+      });
+    });
+
+    describe('ArrowLeft', () => {
+      it('shifts focus to the last tab when pressing ArrowLeft', () => {
+        const tabs = mountWithAppProvider(
+          <Tabs {...mockProps} tabs={mockTabs} />,
+        );
+        trigger(tabs.find('ul'), 'onKeyUp', {
+          key: 'ArrowLeft',
+        });
+        expect(tabs.find(TabMeasurer).prop('tabToFocus')).toBe(2);
+      });
+
+      xit('shifts focus to the first tab when pressing ArrowLeft on the second tab', () => {
```

Comment:
use `it.skip()` if you're going to skip a test.

Or ideally don't have the test at all